### DISCUSSION TITLE
Query all the tables and return the results in an overview table based on a specific column

### DISCUSSION THREAD

**Discussion by BismaIftekhar** [Budibase/budibase#8039] [Help] [answered]
Hello Budibase community,

Is there a way to query all the tables and return the results in an overview table based on a specific column such as Location?

-Using Budibase DB tables

Relational tables : 3
(Physical hosts,VMs, Containers)

Non-relational tables: 3
(Switches, Printers, Access Points)

Results at present:
I have been able to create a relationship table and added a filter based on Location. However, the issue is to add the other three tables (switches, AP,printers) on the same screen.

Desired result: 
One table which shows some data from all the tables based on the column Location.

I am struggling with this problem since days and would really appreciate any help.

Thank you so much.

**Comment by melohagan** [comment, marked as answer]
If the switches and printers tables have no relationship to the other tables, then you won't be able to show them in a single table. That's something you could do with custom queries if using a SQL datasource: https://docs.budibase.com/docs/custom-queries

If you really wanted to, you could create a Many-Many relationship from your main table to the switches and printers tables; that way you can utilise formulas as a method of displaying data from another table: https://docs.budibase.com/docs/formula
That approach could lead to performance consideration however depending on the amount of data involved: https://github.com/Budibase/budibase/discussions/5299

My gut instinct on this one is that you should probably be using a relational database for what you're trying to do.

**Reply by BismaIftekhar** [reply]
Thank you so much for taking out time. Right now I am still trying to work with Budibase DB but as you said, may have to use SQL.
